dotcom-rendering/src/web/server/index.ts

```diff
@@ -184,6 +187,36 @@ export const renderKeyEvents = (
 	}
 };
 
+/**
+ * Sends markup to Frontend, which forwards it to the client.
+ * Frontend expects that it will receive the markup string as a value of the
+ * key 'html' in a json object, i.e. { "html": "<some markup>" }
+ */
+export const renderCards = (
+	{ body }: { body: CardsRequest },
```

Comment:
I tend to think it's best to not make an assumption about the type `body`, and parse it—we use AJV for `enhanceCAPIType` and `enhanceFronts`. We're not very consistent, though.

Reply:
this definitely seems like a good practice to me. The current pattern is (afaict) the same as used by the other endpoints in this file though, so do you think it might make sense to open this as a separate issue, or to roll it into this one?